### DISCUSSION TITLE
A small UI request for prompt and negative prompt fields to adjust attention weights once supported

### DISCUSSION THREAD

**Discussion by cleverestx** [VoltaML/voltaML-fast-stable-diffusion#74] [Ideas]
Assuming weights get supported down the line, being able to highlight (as automatic1111 and Vladmandic builds allow) any word and then hold down CTRL and press the up or down arrow to change the weights of the words would be nice...so if I highlight say, high quality and press CTRL+UP it becomes (high quality1:1), press again and it becomes (high quality1:2), and also down from the start becomes (high quality0:9)

**Comment by Stax124** [comment]
Added in 9739dca
Prompts:
- ArrowUp -> +0.1
- ArrowDown -> -0.1

General:
- ESC -> Interrupt
- CTRL+Enter -> Generate
